cvc5_pythonic_api/cvc5_pythonic.py

```diff
@@ -1727,6 +1734,793 @@ def Or(*args):
     return _nary_kind_builder(Kind.OR, *args)
 
 
+#########################################
+#
+# Strings and Sequences
+#
+#########################################
+
+
+class SeqSortRef(SortRef):
+    def is_string(self):
+        """Determine if sort is a string
+
+        >>> s = StringSort()
+        >>> s.is_string()
+        True
+        >>> s = SeqSort(IntSort())
+        >>> s.is_string()
+        False
+        """
+        return isinstance(self, StringSortRef)
+
+
+class SeqRef(ExprRef):
+    """Sequence Expressions"""
+
+    def sort(self):
+        return _sort(self.ctx, self.ast)
```

Comment:
This is equivalent to the `ExprRef::sort` implementation, correct? If so, we should remove this one.